jaxb-annox-parent/samples/pom.xml

```diff
@@ -116,14 +113,17 @@
 					</execution>
 				</executions>
 			</plugin>
-			<plugin>
-				<groupId>org.apache.maven.plugins</groupId>
-				<artifactId>maven-deploy-plugin</artifactId>
-				<version>2.8.1</version>
-				<configuration>
-					<skip>true</skip>
-				</configuration>
-			</plugin>
 		</plugins>
+		<pluginManagement>
```

Comment:
Make this a property instaed <maven.deploy.skip>true..

Reply:
Done